packages/@aws-cdk/aws-cloudformation/lib/pipeline-actions.ts

```diff
@@ -118,24 +118,24 @@ export interface PipelineCloudFormationDeployActionProps extends PipelineCloudFo
    * IAM role to assume when deploying changes.
    *
    * If not specified, a fresh role is created. The role is created with zero
-   * permissions unless `fullPermissions` is true, in which case the role will have
+   * permissions unless `adminPermissions` is true, in which case the role will have
    * full permissions.
    *
-   * @default A fresh role with full or no permissions (depending on the value of `fullPermissions`).
+   * @default A fresh role with full or no permissions (depending on the value of `adminPermissions`).
    */
   role?: iam.Role;
 
   /**
    * Acknowledge certain changes made as part of deployment
    *
    * For stacks that contain certain resources, explicit acknowledgement that AWS CloudFormation
-   * might create or update those resources. For example, you must specify CAPABILITY_IAM if your
-   * stack template contains AWS Identity and Access Management (IAM) resources. For more
-   * information, see [Acknowledging IAM Resources in AWS CloudFormation Templates](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities).
-   *
-   * @default No capabitilities passed, unless `fullPermissions` is true
+   * might create or update those resources. For example, you must specify `AnonymousIAM` or `NamedIAM`
+   * if your stack template contains AWS Identity and Access Management (IAM) resources. For more
+   * information
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities
```

Comment:
I don't think this will render well (the sentence will be cut off, and the @see will be below the text paragraph. Feel free to use Markdown links here: `For more information, [see the AWS documentation](https://docs.aws.amazon.com/somepath)`.

Reply:
Ok, this way is fine 🙂